src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,43 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const currentTab = tabs.find(({ id }) => id === selectedTabId) || tabs[0].id;
```

Comment:
```suggestion
  const currentTab = tabs.find(({ id }) => id === selectedTabId) || tabs[0];
```
Also, here as well you need to store the tab if it isn't found instead of the id, in your case test failed and you will get an error.

![image](https://github.com/mate-academy/react_tabs-js/assets/63243136/2ff9d15c-6481-4f65-bab3-f7bed9685298)